fuel/app/classes/controller/widgets.php

```diff
@@ -290,7 +291,7 @@ protected function retired()
 		Js::push_group(['angular', 'materia']);
 	}
 
-	protected function no_attempts($inst)
+	protected function no_attempts(string $inst, bool $is_embedded)
```

Comment:
```suggestion
	protected function no_attempts(object $inst, bool $is_embedded)
```
`$inst` is definitely always going to be an object here.

This function is only called from one place: Currently line 390 in this same file, where it is never provided with a second argument, meaning `$is_embedded` is always missing.

That's likely unintended, so line 390 will need to be modified to pass `$is_embedded` as well.

Reply:
Resolved via a58442c9601d6e9692686cb6cbee9407fe819880